osmaxx/excerptexport/models/excerpt.py

```diff
@@ -1,24 +1,34 @@
-from django.db import models
 from django.contrib.auth.models import User
+from django.contrib.gis.db import models
 from django.utils.translation import gettext_lazy as _
 
 
 class Excerpt(models.Model):
     name = models.CharField(max_length=128, verbose_name=_('name'))
     is_public = models.BooleanField(default=False, verbose_name=_('is public'))
     is_active = models.BooleanField(default=True, verbose_name=_('is active'))
-
     owner = models.ForeignKey(User, related_name='excerpts', verbose_name=_('owner'))
-    bounding_geometry = models.OneToOneField('BoundingGeometry', verbose_name=_('bounding geometry'))
+    bounding_geometry = models.MultiPolygonField(verbose_name=_('bounding geometry'), null=True)
+    country = models.ForeignKey('countries.Country', verbose_name=_('Country'), null=True)
```

Comment:
I think `on_delete=CASCADE` is actually the correct behavior: We would have to take the boundary from the country and add it as `bounding_geometry` which can't be achieve without using triggers or similar mechanisms. Otherwise we would have invalid objects in the database, which have neither a country nor a `bounding_geometry` set.